components/Link/index.js

```diff
@@ -0,0 +1,12 @@
+const Link = ({ to, children, header, ...props }) => {
+  if (to) {
+    return (
+      <a href={to} target="_blank" rel="noopener noreferrer" {...props}>
+        {children}
+      </a>
+    );
+  }
+  return <div {...props}>{children}</div>;
```

Comment:
Se usa este escenario?

Reply:
Sí se usa, cuando el no el `to` es null, este componente lo uso en los casos cuando hay información de la fuente, pero no se esta seguro si en enlace existe o no, por lo que el div se usa como un fallback, de tal manera que cuando el `to` sea nulo, entonces no cree un tag a sino un div.

Reply:
Vale, entonces si el componente hace 2 cosas a veces vale la pena indicarlo, que te parece el nombre `LinkWithFallback`?

Reply:
dale, creo que si lo puedo cambiar para que tenga más sentido

Reply:
changed in [247330f](https://github.com/openpolitica/proclama-preliminar/pull/100/commits/247330fbd2c4171c9cca93d48b794e59bd9403fa)

Reply:
lo vi en 4d3f2a68bd3b3bd62ed1bd07328d492225783aab, gracias!